src/models-spec.js

```diff
@@ -103,15 +114,23 @@ describe('models', () => {
               if (!result.pass) {
                 result.message = `${typeRef} is not a valid schema.org reference`;
               }
+            } else if (typeId.match(/^http:\/\/purl\.org\/goodrelations\/v1/)) {
+              result.pass = goodRelationsDataModel.includes(typeId);
+              if (!result.pass) {
+                result.message = `${typeRef} is not a valid goodrelations reference`;
+              }
+            } else if (typeId.match(/^http:\/\/www\.w3\.org\/2004\/02\/skos\/core#/)) {
+              result.pass = skosDataModel.includes(typeId);
+              if (!result.pass) {
+                result.message = `${typeRef} is not a valid goodrelations reference`;
```

Comment:
Is this error message wrong? Should be SKOS?

Reply:
Doh, went a bit blind after starring at too much RDF/XML clearly. Have matched the error messages up in a later PR.

Reply:
Super!